### PR TITLE
Add alt text to icon img tags

### PR DESCRIPTION
**Pull request by harupy**
## What changes are proposed in this pull request?

This PR adds an alt prop to the image icon tags and remove warnings `jsx-a11y/alt-text`.

Before:
Screen Shot 2019-11-07 at 12 55 59

After:
Screen Shot 2019-11-13 at 0 43 14

## How is this patch tested?

Manually tested.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by Zangr** [comment]
Maybe we should actually show some text? @gioa , what do you think about the alternative text to show when image is not available?

**Comment by gioa** [comment]
+1000 on adding an alternative text for all the images for better accessibility!

**Comment by Zangr** [comment]
@gioa , what about the specific text we should add for each of the images?

**Comment by gioa** [comment]
The alternative text usually describes the image. If we describe it as "Laptop, File, Project, and Empty", how will that look? If the alternative text looks bad when the image isn't available, I am okay with an empty string - just as the warning suggests, give an empty string for decorative images.

**Review by Zangr — APPROVED** [review]
@gioa thanks for confirming. This change LGTM.

**Comment by Zangr** [comment]
@harupy before we merge this PR. Could you please help do a bit of search to see if there is other places that this fix can be applied? Let's fix them all at once.

File: mlflow/server/js/.eslintrc.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  'extends': 'airbnb-base',
+  'extends': [
+    'airbnb-base',
+    'react-app',
+  ],
```
Comment (harupy) [inline mlflow/server/js/.eslintrc.js]: Added `react-app` here to make the linter catch `jsx-a11y/alt-txt` warning.

**Comment by harupy** [comment]
@Zangr cc @gioa 
I ran `npm runt lint` and confirmed there is no `jsx-a11y/alt-txt` warning.

**Pull request merged**